packages/dev/create/templates/block/template/test/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Block test.
+ */
+
+import * as Blockly from 'blockly';
+import '../src/index.js';
```

Comment:
Is this the correct style of import?

I also see that we're following our old style of defining the block when you load the file.  Do we have any interest in switching to making block definition more explicit, with an `addBlockDefinition` call or similar that's exported from any block definition module?

Reply:
Since nothing is exported when you define a block. (We only really need the file to be included). That style of import is correct. (Giving it a name would lead to an unused variable).

As for how we want us / developers to develop custom blocks, I have no real opinion on this.

Reply:
I thought about it for a while and forcing an extra init step per block file seems unnecessary, so I'm fine with leaving it the way it is.